crates/invoker-impl/src/lib.rs

```diff
@@ -64,6 +64,8 @@ use crate::metric_definitions::{
     TASK_OP_SUSPENDED,
 };
 
+const RESTATE_SERVICE_PROTOCOL_VERSION: u16 = 1;
```

Comment:
Should this maybe go into the `service-protocol` crate to be closer to the service protocol definition?